client/src/AddVideo.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+
+function AddVideo(props) {
+    const [videoTitleInput, setVideoTitleInput] = useState("");
+    const [videoUrlInput, setVideoUrlInput] = useState("");
+
+    function handleTitleChange(event) {
+      setVideoTitleInput(event.target.value);
+    }
+
+    function handleUrlChange(event) {
+      setVideoUrlInput(event.target.value);
+    }
+
+    function addVideoObj() {
+      const videoObj = {title: videoTitleInput, url: videoUrlInput};
+      props.AddVideo(videoObj);
```

Comment:
I think `addVideo` here is a function, not a component, so it should be a lowercase `a` at the beginning